pylintrc

```diff
@@ -0,0 +1,8 @@
+[BASIC]
+
+# Regular expression matching correct method names
+method-rgx=[a-z_][a-z0-9_]{2,40}$
```

Comment:
My tests methods are longer than 30 chars, and I like them that way. I can do a local disable if you prefer, let me know.